### PR TITLE
[rfc] onednn integration via byoc

### PR DESCRIPTION
**Pull request by crazydemo**
add rfc: oneDNN integration via BYOC

### CODE REVIEW

File: rfcs/0042-onednn-integration-via-byoc.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- Depthwise *
+- Binary *
+- PReLu *
+
+## Datatype
+- Float32
+- BF16 *
+- INT8 *
+
+## Verified CV Models (from gluoncv)
```
Comment (ZhennanQin) [inline rfcs/0042-onednn-integration-via-byoc.md]: Bert is not a CV models. Let's remove `CV`

File: rfcs/0042-onednn-integration-via-byoc.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- ResNet 18, 32, 50, 101, 152
+- VGG 11, 13, 16, 19; VGG_BN 11, 13, 16, 19
+- InceptionV3
+- Mobilenet *
+- Bert *
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Currently, only test on CPU.
```
Comment (ZhennanQin) [inline rfcs/0042-onednn-integration-via-byoc.md]: only test on `Intel` CPU.

File: rfcs/0042-onednn-integration-via-byoc.md (hunk 1 above)
Comment (crazydemo) [reply rfcs/0042-onednn-integration-via-byoc.md]: modified.

File: rfcs/0042-onednn-integration-via-byoc.md (hunk 2 above)
Comment (crazydemo) [reply rfcs/0042-onednn-integration-via-byoc.md]: modified.

**Pull request closed without merging**